plugin/service.go

```diff
@@ -82,6 +65,12 @@ func (s *PluginManager) getPlugin(name PluginInterfaceName) (managedPlugin, bool
 	return p, ok
 }
 
+// Check if a plugin is enabled/setup
+func (s *PluginManager) IsEnabled(name PluginInterfaceName) bool {
```

Comment:
Is this function being used. I don't see any references of this

Reply:
ah, this is a place holder. It will be used in the coming PRs.

Reply:
Sounds good